docs/import-roster-from-lms.md

```diff
@@ -0,0 +1,78 @@
+## Importing a course roster from your Learning Management System
+
+GitHub Classroom is able to automatically import your course roster from your institution's Learning Management
+System, potentially saving you time and headaches when provisioning a new GitHub Classroom course!
+
+### Prerequisites
+
+This guide assumes that you've successfully [connected GitHub Classroom to you Learning Management System](/help/connect-to-lms). If you haven't, please follow those setup instructions before proceeding to the next steps.
+
+Importing a course roster from your Learning Management System will only work if you have you have enabled GitHub Classroom to receive course roster information per the privacy settings within your Learning Management System.
+
+### Creating a new course roster
+
+To import a new course roster from your Learning Management, select the classroom you would like to import into. Then proceed through the following steps:
```

Comment:
I'm going to suggest reworking the introduction here and initial steps here. If you need the reader to take some action, it may as well be a numbered step. Kinda like this:

> To import a course roster from your learning management system:
> 
> 1. [Sign in to GitHub Classroom](https://classroom.github.com/login).
> 2. [step(s) here to click the course and get to the right settings page]

I have some more tips that apply to this whole section below and the syncing section